source/shared/cpp/AdaptiveCardsSharedModel/AdaptiveCardsSharedModelUnitTest/DateAndTimeUnitTest.cpp

```diff
@@ -1,6 +1,7 @@
 #include "stdafx.h"
 #include "CppUnitTest.h"
 #include "TextBlock.h"
+#include "MarkDownParser.h"
```

Comment:
>MarkDownParser [](start = 10, length = 14)

Is this necessary?

Reply:
Thanks!

---
In reply to: [158142017](https://github.com/Microsoft/AdaptiveCards/pull/1053#discussion_r158142017) [](ancestors = 158142017)